docs/src/pages/guides/notifications.md

```diff
@@ -9,7 +9,7 @@ In `Monika`, we provide multiple notification channels. Notifications are availa
 
 ## Configurations
 
-Notifications can be sent through multiple channels and mail accounts, each are defined through configuration such as in the file `config.json.example` below. All you need for having a notification is set these configurations.
+Notifications can be sent through multiple channels and mail accounts, each are defined through configuration such as in the file `config.json.example` below.
```

Comment:
Notification could be sent through multiple channels and emails. You should define notification mechanism in the configuration as shown in `config.json.example` bellow.

Reply:
it's `config.example.json` now

Reply:
updated, please check